### PR TITLE
feat(es/minifier): Inline uninitialized vars with sequential inliner

### PR DESCRIPTION
**Pull request by kdy1**
**Description:**

An uninitialized variable is inlinable.

**Related issue:**

### CODE REVIEW

File: crates/swc_ecma_minifier/tests/benches-full/victory.js [hunk 1]
```diff
@@ -25305,15 +25305,15 @@
                 if (n = (values = Float64Array.from(Object(_number_js__WEBPACK_IMPORTED_MODULE_3__.numbers)(values, valueof))).length) {
                     if ((p = +p) <= 0 || n < 2) return Object(_min_js__WEBPACK_IMPORTED_MODULE_1__.default)(values);
                     if (p >= 1) return Object(_max_js__WEBPACK_IMPORTED_MODULE_0__.default)(values);
-                    var n, i = (n - 1) * p, i0 = Math.floor(i), value0 = Object(_max_js__WEBPACK_IMPORTED_MODULE_0__.default)(Object(_quickselect_js__WEBPACK_IMPORTED_MODULE_2__.default)(values, i0).subarray(0, i0 + 1));
```
Comment (kdy1) [inline crates/swc_ecma_minifier/tests/benches-full/victory.js:25308]: This seems wrong, so `reassigned()` is wrong I guess

**Review by kdy1 — COMMENTED** [review]
swc-bump:
 - swc_ecma_minifier

**Review by kdy1 — COMMENTED** [review]
This PR is for feature parity with terser and future optimizations

**Comment by Austaras** [comment]
I don't believe seq inliner is the proper place for this. Unintialized vars should be inlined anywhere as accessing them are always side effect free. However this doesn't really matter as it's very rare to have unintialized vars in real code.

**Comment by kdy1** [comment]
You are right, nice catch!
I'll change it to use inliner instead

**Comment by kdy1** [comment]
I have thought about this, and I concluded this is useless

**Pull request closed without merging**